### PR TITLE
Add arm32v7 alpine 3.9 Dockerfile

### PR DESCRIPTION
**Pull request by krwq**
cc: @janvorli 

except for numactl-dev missing (replaced with building from sources), it is a copy of arm64 dockerfile

### CODE REVIEW

**Comment by mthalman** [comment]
This new Dockerfile needs to be referenced in the manifest.json file at the root of the repo in order for it to get built.  Add this snippet next to the existing src/alpine/3.9/arm64v8 reference:
```
        {
          "platforms": [
            {
              "architecture": "arm",
              "dockerfile": "src/alpine/3.9/arm32v7",
              "os": "linux",
              "tags": {
                "alpine-3.9-arm32v7-$(System:DockerfileGitCommitSha)-$(System:TimeStamp)": {}
              },
              "variant": "v7"
            }
          ]
        },
```

**Comment by janvorli** [comment]
@krwq the numactl-dev is not needed for arm32, you can just remove it. We do the same for linux-arm. Enabling NUMA on 32 bit systems doesn't make sense.

**Comment by krwq** [comment]
@janvorli, thanks, I didn't know what NUMA is

File: manifest.json [hunk 1]
```diff
@@ -683,6 +683,19 @@
...
+              "architecture": "arm",
+              "dockerfile": "src/alpine/3.9/arm32v7",
+              "os": "linux",
+              "tags": {
+                "alpine-3.9-arm32v7-$(System:DockerfileGitCommitSha)-$(System:TimeStamp)": {}
+              },
+              "variant": "v7"
+            }
+          ]
+        },
```
Comment (mthalman) [inline manifest.json]: Nit: Could you actually move this up above the 3.9 ARM64 entry?  That'll keep things consistent where ARM32 is listed before ARM64 for everything.

**Comment by mthalman** [comment]
Failed build looks to be a random failure unrelated to the change.  I'll go ahead and merge the changes.

**Comment by mthalman** [comment]
Images produced by this change:

mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.9-arm32v7-2612d91-20190613183347

**Pull request merged**